libs/sdk-ui-ext/src/internal/utils/colors.ts

```diff
@@ -18,11 +18,11 @@ import {
     isMeasureDescriptor,
     isResultAttributeHeader,
 } from "@gooddata/sdk-model";
-import { getMappingHeaderName, IColorAssignment, IMappingHeader } from "@gooddata/sdk-ui";
+import { getMappingHeaderFormattedName, IColorAssignment, IMappingHeader } from "@gooddata/sdk-ui";
 import { ColorUtils } from "@gooddata/sdk-ui-charts";
 
 function getItemName(item: IColoredItem): string {
-    return getMappingHeaderName(item.mappingHeader) || "";
+    return getMappingHeaderFormattedName(item.mappingHeader) || "";
```

Comment:
This is not needed anymore and was part of my changes for execution. Looks like some leftovers after rebase from the past.